mobile_app/make_build.py

```diff
@@ -80,10 +85,36 @@ def collect_params():
     return (args, environ)
 
 
+def make_build(code_repo, trigger_repo):
+    """
+    Checks out the trigger repo then makes a new
+    build with remaining arguments supplied by the user
+    """
+    trigger_path_container = tempfile.mkdtemp()
+    # Filter to the questions we're not prepopulating
+    questions = [
+        question for question in QUESTIONS if
+        question.key != 'CODE_REPO' and
+        question.key != '--trigger-repo-path'
+    ]
+
+    try:
+        (args, environ) = collect_params(questions)
+
+        environ["CODE_REPO"] = code_repo
```

Comment:
sorry ignorant question...but if I have already set my shell's envrionment variable, it won't get changed here, will it? NBD since it needs to be supplied the script (and therefore I probably wouldn't set my shell env variable anyway), but I was curious.